db/schema.rb

```diff
@@ -728,4 +728,34 @@
   add_foreign_key "vita_partner_zip_codes", "vita_partners"
   add_foreign_key "vita_partners", "coalitions"
   add_foreign_key "vita_providers", "provider_scrapes", column: "last_scrape_id"
+
+  create_view "organization_capacities", sql_definition: <<-SQL
+      WITH organization_id_by_vita_partner_id AS (
+           SELECT vita_partners_1.id,
+                  CASE
+                      WHEN (vita_partners_1.parent_organization_id IS NULL) THEN vita_partners_1.id
+                      ELSE vita_partners_1.parent_organization_id
+                  END AS organization_id
+             FROM vita_partners vita_partners_1
+          ), client_ids AS (
+           SELECT DISTINCT tax_returns.client_id
+             FROM tax_returns
+            WHERE ((tax_returns.status > 100) AND (tax_returns.status < 405))
+          ), partner_and_client_count AS (
```

Comment:
Trivial, but now that I think about it, in Rails, table names are usually plural, so consider `partner_and_client_counts`. Also consider `client_count_by_organization_id`. I love using `x_by_y` in names.